NuGet.Config

```diff
@@ -3,6 +3,7 @@
   <packageSources>
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
+    <add key="templating" value="https://dotnet.myget.org/F/templating/api/v3/index.json" />
```

Comment:
Which feeds are these packages coming from? We may need to setup the process to have the packages propagated.